permissions/plugin-http_request.yml

```diff
@@ -6,6 +6,4 @@ issues:
 paths:
   - "org/jenkins-ci/plugins/http_request"
 developers:
-  - "janario"
-  - "markewaite"
```

Comment:
I need to be retained as a plugin maintainer because I can help with tasks as needed.

Reply:
The typical pattern is to add yourself as a maintainer but to not remove other maintainers.

Reply:
Of course, no problem there. I assumed none of you want to keep maintaining it, because it was for adoption.

Reply:
@MarkEWaite I retained previous developers. New commit is added.